crates/sui-indexer/migrations/2023-04-01-211603_drop_events_parsed_json/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE events
+    DROP COLUMN IF EXISTS parsed_json;
```

Comment:
do we have an option of running this extra migration but not resetting DB?
otherwise might need to do this manually?

Reply:
need to run diesel migration, there are ways to do this programatically, not sure if we have that or not, it's similar to reset but without dropping table/reverting migration.

for now I guess we just need to do `diesel migration run ...`